swarm/network/pss_error.go

```diff
@@ -0,0 +1,32 @@
+package network
+
+import (
+	"fmt"
+)
+
+const (
+	pssError_NoForwarder = iota
+	pssError_ForwardToSelf
+	pssError_BlockByCache
+)
+
+var pssErrorToString = map[int]string{
+	pssError_NoForwarder:   "no available forwarders in routing table",
```

Comment:
the usual way is to simply use the `errors` packege

```
errPssForwardSelf = errors.New("forward to self")
```
just add these in a var()  section on top of pss.go, no need for separate file